### PR TITLE
Include release notes for 1.10.0.0

### PR DESCRIPTION
**Pull request by VijayanB**
Added release notes with categories and corresponding PR.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: release-notes/opendistro-for-elasticsearch-anomaly-detection.release-notes-1.10.0.0.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+## Version 1.10.0.0 Release Notes
```
Comment (kaituo) [inline release-notes/opendistro-for-elasticsearch-anomaly-detection.release-notes-1.10.0.0.md:1]: An observation from an ODFE user (posted on our discussion forum), the download links used in plugin packages are incorrect. Requesting all plugin owners to correct it, if artifacts not published yet.
What it is:
￼
  https://opendistro.github.io/elasticsearch/downloads
What it should be:
￼
  https://opendistro.github.io/for-elasticsearch/downloads.html
Where to change: In build.gradle file
Example:  https://github.com/opendistro-for-elasticsearch/anomaly-detection/blob/master/build.gradle#L347
 
Can you modify as mentioned above in your release PR?

Comment (VijayanB) [reply release-notes/opendistro-for-elasticsearch-anomaly-detection.release-notes-1.10.0.0.md:1]: Yeah sure.

File: release-notes/opendistro-for-elasticsearch-anomaly-detection.release-notes-1.10.0.0.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Compatible with Elasticsearch 7.9.0
+
+### Features
+
+ * AD CLI ([#196](https://github.com/opendistro-for-elasticsearch/anomaly-detection/pull/196))
```
Comment (ylwu-amzn) [inline release-notes/opendistro-for-elasticsearch-anomaly-detection.release-notes-1.10.0.0.md:7]: So we are going to release AD CLI along with 1.10?

File: release-notes/opendistro-for-elasticsearch-anomaly-detection.release-notes-1.10.0.0.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+ * Get detector ([#207](https://github.com/opendistro-for-elasticsearch/anomaly-detection/pull/207))
```
Comment (ylwu-amzn) [inline release-notes/opendistro-for-elasticsearch-anomaly-detection.release-notes-1.10.0.0.md:8]: How about we make the description more clear ? Like "Get detector command in AD CLI tool"

**Pull request merged**